spec/Cabin/SingleSeat.vspec

```diff
@@ -53,14 +53,12 @@ Massage:
   description: Seat massage level. 0 = off. 100 = max massage.
 
 Recline:
```

Comment:
Recline of what - whole seat, seating/cushion or backrest?

I'd prefer longer and more unique names instead of using synonyms for different elements of the seat which could have a recline/tilt.
"Short names" should only refer to the seat as a whole, like longitudinal position, height, tilt.

Proposal: Maybe have a sub-branch for the backrest (like for the cushion/seating).